features/steps/packages.py

```diff
@@ -37,6 +37,39 @@ def when_i_apt_update(context, machine_name=SUT):
     )
 
 
+@when("I perform a {dry} run of {command} upgrade")
+@when("I upgrade all apt packages in the system")
+@when("I upgrade all apt packages in the system using {command}")
+@when("I upgrade all apt packages in the system including {phased} updates")
```

Comment:
Do we ever not want to include phased updates?

Reply:
I would say the standard is to not include them, so we can consider acceptance tests to do what users do.
We install phased updates when testing our apt messages, for practicality purposes.